src/components/License/VLicense.vue

```diff
@@ -72,37 +75,30 @@ export default {
     const isCcLicense = computed(() => isCc(props.license))
 
     const icons = computed(() => {
-      let iconList = props.license.split(/[-\s]/)
+      let iconList: (LicenseElement | 'ccLogo')[] = props.license.split(
+        /[-\s]/
+      ) as LicenseElement[]
       if (isCcLicense.value) iconList = ['ccLogo', ...iconList]
       return iconList
```

Comment:
This mutable array reference pains me! I think we can get rid of some of the painful type juggling here by treating `iconList` as a `const`, if you're interested:

```
      const iconList = props.license.split(/[-\s]/) as LicenseElement[]
      if (isCcLicense.value) return ['ccLogo', ...iconList] as const
      return iconList
```

Take it or leave it though :slightly_smiling_face: